CHANGELOG.md

```diff
@@ -42,6 +42,9 @@ https://qooxdoo.org/documentation/#/development/compiler/migration .
 
 ## Breaking changes
 
+- `qx.ui.core.scroll.AbstractScrollArea`: The width and height are now fix. 
```

Comment:
s/fix/fixed/

Reply:
What does "fixed" mean in this context? That the scroll area doesn't grow or shrink? Or that they need to be set with absolute values?